src/components/TodoForm/index.jsx

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import { Input } from "reactstrap";
+import { todosAction } from "../../redux/actions";
 
-import styles from './styles.module.sass';
+import styles from "./styles.module.sass";
+
+function TodoForm({ addTodo }) {
+  const [todo, setTodo] = useState("");
+
+  const saveTodo = (e) => {
+    if (e.keyCode === 13 && todo !== "") {
+      addTodo(todo);
+      setTodo("");
+    }
+  };
 
-function TodoForm() {
   return (
-    <form>
-      <input
+    <>
+      <Input
         type="text"
-        placeholder="What do you have to do?"
+        placeholder="What do you have to do? (Press Enter to add todo)"
         className={styles.input}
+        onChange={(e) => setTodo(e.target.value)}
+        onKeyUp={saveTodo}
```

Comment:
Using `onKeyUp`fires an event even if the key was originally held down outside the component. use `onKeyPress` instead